### ISSUE TITLE
Interpretation of score

### ISSUE THREAD

**Issue by DecentMakeover** [MIC-DKFZ/nnUNet#49]
Hi 

I Get this score 

                2019-08-28 07:32:10.274085:
		epoch:  519
		2019-08-28 07:38:44.771640: train loss : -0.9634
		2019-08-28 07:39:09.508282: val loss (train=False): -0.0622
		2019-08-28 07:39:09.508541: Val glob dc per class: [0.07488450598148061]
		2019-08-28 07:39:10.148665: lr is now (scheduler) 1.2e-05
		2019-08-28 07:39:10.148762: current best_val_eval_criterion_MA is 0.08170
		2019-08-28 07:39:10.148827: current val_eval_criterion_MA is 0.0713
		2019-08-28 07:39:10.148878: No improvement: current train MA -0.9654, best: -0.9663, eps is 0.0005
		2019-08-28 07:39:10.148948: Patience: 19/50
		2019-08-28 07:39:10.149020: This epoch took 419.234268 s

		2019-08-28 07:39:10.149094:
		epoch:  520
		2019-08-28 07:45:44.826182: train loss : -0.9671
		2019-08-28 07:46:09.554083: val loss (train=False): -0.0630
		2019-08-28 07:46:09.554411: Val glob dc per class: [0.07556941617334902]
		2019-08-28 07:46:10.196557: lr is now (scheduler) 1.2e-05
		2019-08-28 07:46:10.196655: current best_val_eval_criterion_MA is 0.08170
		2019-08-28 07:46:10.196733: current val_eval_criterion_MA is 0.0717
		2019-08-28 07:46:10.196869: No improvement: current train MA -0.9655, best: -0.9663, eps is 0.0005
		2019-08-28 07:46:10.196955: Patience: 20/50
		2019-08-28 07:46:10.197019: This epoch took 419.405029 s

As you see the train loss is -96, but the val loss is nowhere close, i haven't  predicted on this model yet as it is still training, but does the score make sense to you ?

Thanks.

**Comment by FabianIsensee** [comment]
Hi,
this looks like you are overfitting like crazy. What data are you running this on? Are you sure everything is alright with your dataset?
Best,
Fabian

**Comment by DecentMakeover** [comment]
Hey sorry was caught up for a while, this was one of struct seg dataset, lung.

Thanks

**Comment by FabianIsensee** [comment]
I don't know what is wrong to be honest. i am participating in StructSeg as well and I don't have this issue

**Comment by DecentMakeover** [comment]
Oh , then I think I must have done something wrong when preparing the dataset

Good luck with structSeg

On 24-Sep-2019, at 4:49 PM, Fabian Isensee <notifications@github.com<mailto:notifications@github.com>> wrote:


I don't know what is wrong to be honest. i am participating in StructSeg as well and I don't have this issue

**Comment by DecentMakeover** [comment]
Congrats on the StructSeg win.
